CONTRIBUTING.md

```diff
@@ -132,45 +132,92 @@ If an issue is a `bug`, and it doesn't have a clear reproduction that you have p
 
 > If you want to work on a UI feature, refer to the [Storybook UI](https://github.com/storybooks/storybook/tree/master/lib/ui) page.
 
-This project written in ES2016+ syntax so, we need to transpile it before use.
-So run the following command:
+### Prerequisites
 
-```sh
-npm run dev
-```
+Please have the **_latest_** stable versions of the following on your machine
 
-This will watch files and transpile in watch mode.
+-   node
+-   yarn
+-   @storybook/cli
```

Comment:
Not really needed

Reply:
@storybook/cli or the whole Prerequisits section?

Reply:
@ storybook / cli or the whole prerequisites section?

Reply:
The `cli` package. You either use example app, or your own, which already has storybook

Reply:
👍